### ISSUE TITLE
Push notifications on Lineage OS and MicroG

### ISSUE THREAD

**Issue by josephtocci** [vector-im/riot-android#1833]
I switched from iPhone to Android, and I installed LineageOS with MicroG on my phone. I turned on GCM in the MicroG settings, and I get push notifications from other apps, but not Riot. My iPhone is still a notification target for Riot, but my Android has not been added. I have submitted a bug report through the app so you can have the logs.

How do I fix the issue?

**Comment by ArchangeGabriel** [comment]
Where did you get Riot from? F-Droid or Play Store? F-Droid version does not have push notifications, only Play Store one.

**Comment by josephtocci** [comment]
I used the F-Droid one, I checked both descriptions and I think the warning should be higher in the description. I might've noticed it if it was the first or the second sentence. A similar warning that there is a more secure riot on F-Droid would be nice on the Play Store.

But that's just nitpicking so ignore me.

Using the Google Play Store version from the Yalp store fixed my issue. Thank you and Happy New Year!
